skynet/env.py

```diff
@@ -1,12 +1,27 @@
 import os
 import sys
 import uuid
+from functools import lru_cache
+
+import torch
 
 app_uuid = str(uuid.uuid4())
 
 is_mac = sys.platform == 'darwin'
 
 
+@lru_cache(maxsize=100)
+def get_device() -> str:
+    if torch.cuda.is_available():
+        print('CUDA device found.')
```

Comment:
Use the logger?

Reply:
didnt want to create circular deps, is that a thing to worry about?